spec/models/contact_details_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+describe ContactDetails, type: :model do
+  it { is_expected.to validate_presence_of(:phone_number).on(:update) }
```

Comment:
Now we've separated the model, we can test validation for all cases, not just `update`?

```suggestion
  it { is_expected.to validate_presence_of(:phone_number) }
```